### DISCUSSION TITLE
v2024.2.0

### DISCUSSION THREAD

**Discussion by Gerben321** [Gerben321/bike-care-tracker#19] [Announcements]
## New features
- New design! 🥳 I'm super happy to announce the new design is ready for the public to test. It now includes a dark mode and much better navigation overall.
- Dynamic form option for adding/editing service schedules. The form options change based on your input, and the values and helpers change accordingly. 
- The dropdown for component type is now an autocomplete field. We have added a lot of types of components, so to make selecting one easier you can type 'bearing' to see all results.
- Option for distance and height unit
- Option for weight unit
- Option for decimal separator
- Option for currency
- Option to add your email address for email notifications
- Notification center in dashboard
- Clickable push notifications on mobile and desktop (tested on Android and Windows 11)

## Fixed
- Solved some `division by zero` issues that users came across when adding/editing service schedules.
- Service schedule cannot be removed (https://github.com/Gerben321/bike-care-tracker/issues/17)
- Components can't be removed (https://github.com/Gerben321/bike-care-tracker/issues/15)
- Translation improvements (https://github.com/Gerben321/bike-care-tracker/issues/13)
- Add purchase date and price to bike (https://github.com/Gerben321/bike-care-tracker/issues/12)
- Fix inconsistent date notation (https://github.com/Gerben321/bike-care-tracker/issues/10)
- End of component usage should default to 23:59 (https://github.com/Gerben321/bike-care-tracker/issues/9)
- Probably lots of other stuff! 

Have you found any bugs in this release, found things that can be improved, got new ideas? Let me know!

Thanks to @OleksiyRudenko and @Konstantin-Levin  for testing!

<hr /><em>This discussion was created from the release <a href='https://github.com/Gerben321/bike-care-tracker/releases/tag/release'>v2024.2.0</a>.</em>

**Comment by OleksiyRudenko** [comment]
That's a one massive update! Congrats!
I am about to change a part of the drivetrain, so looking forward to use the full potential of the app
